### PR TITLE
add cron for incoming email check via IMAP

### PR DESCRIPTION
**Pull request by machisuji**
Addresses #158 

to do

- [x] allow using existing secret for IMAP credentials
- [x] address /app/docker/cron script in slim container issue

### CODE REVIEW

File: charts/openproject/templates/cron-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+          {{- include "openproject.containerSecurityContext" . | indent 10 }}
+          image: {{ include "openproject.image" . }}
+          imagePullPolicy: {{ .Values.image.imagePullPolicy }}
+          envFrom:
+            {{- include "openproject.envFrom" . | nindent 12 }}
+            - secretRef:
+                name: {{ include "common.names.fullname" . }}-cron-environment
+          command:
+            - bash
+            - /app/docker/prod/cron
```
Comment (oliverguenther) [inline charts/openproject/templates/cron-deployment.yaml:84]: This will only be present in the AIO dockerfile however

Comment (machisuji) [reply charts/openproject/templates/cron-deployment.yaml:84]: Good catch! Maybe we should add it to the slim build too? It's just one script after all.

Comment (machisuji) [reply charts/openproject/templates/cron-deployment.yaml:84]: Checked it and no, the script is present both in AIO and slim containers:

```bash
docker run --rm -it openproject/openproject:15-slim bash -c 'ls /app/docker/prod/cron'
/app/docker/prod/cron
```

So all good here.

**Comment by machisuji** [comment]
@oliverguenther This should be good now.

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**